Portal/src/Datahub.Functions/ProjectUsageScheduler.cs

```diff
@@ -47,28 +65,59 @@ await foreach (var resource in ctx.Project_Resources2.AsAsyncEnumerable())
 
             // send/post the message
             await _mediator.Send(message);
+
+            timeout += 5;
+
+            var capacityMessage = ConvertToCapacityUpdateMessage(message, timeout);
+
+            // send/post the message
+            await _mediator.Send(capacityMessage);
         }
 
         _logger.LogInformation($"{scheduled.Count} projects scheduled!");
     }
 
-    record DBResourceContent(string resource_group_name);
-
-    private ProjectUsageUpdateMessage? TryDeserializeMessage(Project_Resources2 row, int timeout)
+    private ProjectUsageUpdateMessage? TryDeserializeMessage(ProjectResourceData resource, int timeout)
     {
         try
         {
-            var content = JsonSerializer.Deserialize<DBResourceContent>(row.JsonContent);
+            var content = JsonSerializer.Deserialize<DBResourceContent>(resource.JsonContent);
             if (string.IsNullOrEmpty(content?.resource_group_name))
                 return default;
 
-            return new ProjectUsageUpdateMessage(row.ProjectId, content.resource_group_name, timeout);
+            return new ProjectUsageUpdateMessage(resource.ProjectId, content.resource_group_name, resource.Databricks, timeout);
         }
         catch
         {
-            _logger.LogInformation($"Found project {row.ProjectId} with invalid JsonContent:\n{row.JsonContent}");
+            _logger.LogInformation($"Found project {resource.ProjectId} with invalid JsonContent:\n{resource.JsonContent}");
             return default;
         }
     }    
+
+    private async Task<List<ProjectResourceData>> GetProjectResources(DatahubProjectDBContext ctx)
+    {
+        var databrickProjects = new HashSet<int>();
+        var projects = new List<Project_Resources2>();
+        await foreach (var res in ctx.Project_Resources2.AsAsyncEnumerable())
+        {
+            if (res.ResourceType == "terraform:azure-databricks")
+            {
+                databrickProjects.Add(res.ProjectId);
+            }
+            if (res.ResourceType == "terraform:azure-storage-blob")
+            {
+                projects.Add(res);
+            }
```

Comment:
I think there's a method somewhere to build these instead of putting the string literal